src/github/entities/user-installation.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserInstallation extends Base {
+  @Column({ type: 'varchar', length: 20 })
+  installationId: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  organizationName: string;
+
+  @ManyToOne(() => User, (user) => user.installations)
+  user: User;
+}
```

Comment:
PullProofAI

> Note: This review was outside of the patch, but no patch was found that overlapped with it. Original lines [24-24]

The relationship between `UserInstallation` and `User` seems fine. No issues found.


<!-- This is an auto-generated comment by PullProofAI -->